src/platforms/javascript/guides/react/components/errorboundary.mdx

```diff
@@ -45,7 +45,7 @@ function FallbackComponent() {
 class App extends React.Component {
   render() {
     return (
-      <Sentry.ErrorBoundary fallback={FallbackComponent} showDialog>
+      <Sentry.ErrorBoundary fallback={<FallbackComponent />} showDialog>
```

Comment:
```suggestion
      <Sentry.ErrorBoundary fallback={myFallback} showDialog>
```

or

```suggestion
      <Sentry.ErrorBoundary fallback={showThisInstead} showDialog>
```

Reply:
@lobsterkatie Works for me. I updated the example to use a variable (and show what you might assign that variable to). Let me know what you think.

Reply:
Nice. I like the examples.